charts/k8s-service/templates/horizontalpodautoscaler.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.horizontalPodAutoscaler.enabled }}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "k8s-service.fullname" . }}
+  namespace: {{ $.Release.Namespace }}
+spec:
```

Comment:
Would we need a `scaleTargetRef` here to know what we're scaling?

Reply:
Would it make sense to auto-fill the values like this:
```
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: {{ include "k8s-service.fullname" . }}
```
instead of passing those via `{{ .Values.horizontalPodAutoscaler.scaleTargetRef }}`?

Reply:
Did I get the `include` right, @yorinasub17 ? 😄

Reply:
Yup that is correct!